mubench.cli/README.md

```diff
@@ -41,3 +44,40 @@ It supports two run modes:
 In either mode, the `DetectorArgs` provide all input as both the Java source code and the corresponding Bytecode. Additionally, it provides the classpath of all the code's dependencies.
 
 The `DetectorOutput` is essentially a collection where you add your detector's findings, specifying their file and method location and any other property that may assist manuel reviews of the findings. MUBench expects you to add the findings ordered by the detector's confidence, descending. You may also output general statistics about the detector run.
+
+## Releases.yml
+
+This file must be at `detectors/mydetector/releases.yml` and contain a list of releases of your detector.
+
+Entries look like this:
+
+```yaml
+- cli-version: 0.0.10
+  md5: 2470067fac02ee118b7d49287246e20a
+- cli-version: 0.0.8
+  md5: 9e5252816faecf552464f0a6abde714f
+  tag: my-tag
+```
+
+The must contain at least one entry. By default, MUBench uses the newest version listed. Each entry consists of the following keys:
+
+* `cli-version` - The [MUBench Runner](#runner) version implemented by the respective detector release. This information is used to invoke your detector.
+* `md5` - The md5 hash of the `detector/mydetector/mydetector.jar` file. MUBench will use this value to check the integrity of the detector, if it was loaded from a remote source.
```

Comment:
Is it true that we check the md5 only if the detector.jar is downloaded? It actually makes sense to do it this way, but I seem to recall that we currently fail if the md5 is missing or wrong, even if the detector.jar is already in the folder...